test/verify/check-networkmanager-wireguard

```diff
@@ -123,6 +123,24 @@ class TestWireGuard(packagelib.PackageCase, netlib.NetworkCase):
         b.wait_not_present("#network-wireguard-settings-dialog")
         b.wait_in_text(f"#networking-interfaces th:contains('{iface_name}') + td", f"1.2.3.4/32, {m1_ip4}/24")
 
+        # if some wg properties are not valid, for example, if it was changed by some external tool, don't crash
+        m1.execute("sed -i 's/.*allowed-ips.*//' /etc/NetworkManager/system-connections/con-wg0.nmconnection")
```

Comment:
A more canonical way to write this is `'/allowed-ips/d'`, i.e. "delete a line if it contains that string".

Reply:
ah, this looks much better, thanks!